Digipost.Api.Client.Common/Extensions/EnumExtensions.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 using Digipost.Api.Client.Common.Enums;
-using Digipost.Api.Client.Scripts.Xsd.XsdToCode.Code;
+
```

Comment:
Hvis du høyreklikker på `Solution-katalogen` øverst i trestrukturen og velger `Cleanup Code` så rydder den opp i sånne tomme mellomrom og slikt. Det hender den slår seg vrang på andre ting man ikke vil ha med, da kan du ta det på en underkatalog for å ikke gjøre det på hele prosjektet. Vi har hatt for vane å gjøre det på alt før merge siden vi har confet den ganske godt opp, men av og til gjør den noe man ikke helt forventer.